contracts/StablecoinConverter.sol

```diff
@@ -584,7 +596,6 @@ contract StablecoinConverter is EpochTokenLocker {
       */
     function getTradedAmounts(uint128 volume, Order memory order) private view returns (uint128, uint128) {
         uint128 executedBuyAmount = volume;
-        require(currentPrices[order.sellToken] != 0, "prices are not allowed to be zero");
         uint128 executedSellAmount = getExecutedSellAmount(
```

Comment:
I think, we still need to check that the executedSellamounts are bigger than AMOUNT_MINIMUM

Reply:
Let me direct you to the line of code that reads as follows:

```
require(executedSellAmount > AMOUNT_MINIMUM, "Can't sell less than AMOUNT_MINIMUM");
```